python/paddle/nn/layer/common.py

```diff
@@ -915,7 +922,8 @@ class Pad1D(layers.Layer):
     If mode is 'reflect', pad[0] and pad[1] must be no greater than width-1.
 
     Parameters:
-        padding (Tensor | List[int32]): The padding size with data type int32. [len(padding)/2] dimensions
+        padding (Tensor | List[int32], int): The padding size with data type int32. If is int, use the
```

Comment:
unrelated to current PR but `int32` does not seem correct, by default python int is int64

Reply:
done, thx~